ci.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# This file is a "runtime" part from a builder-pattern in Dockerfile, it's used in CI.
```

Comment:
To clarify, what's going on here is that we're taking the binaries built by the CI pipeline and putting them inside a Dockerfile, whose image we then publish?

Where does this leave our top level Dockerfile then?

Reply:
Yeah, exactly.
Top-level `./Dockerfile` is a CI on its own with that docker builder pattern. It's not beneficial to use such a pattern within another CI system as it won't reuse cache normally.
However, it's great for local usage.
I couldn't come up with the idea on how to inherit one from another to stay DRY, there's a `--target` in `docker build` which can choose the exact step of the builder pattern to start from, but
- the unavoidable `COPY` instruction will trigger the initial step anyway, hence it won't help
- it's not yet working in buldah man_shrugging

Reply:
I think we'll need to note down somewhere that our published images don't match that of the `Dockerfile`. I'm afraid it could cause some confusion if people use different images without realizing